services/stripe-billing/lib/index.js

```diff
@@ -74,7 +76,13 @@ module.exports = class StripeBillingService extends CampsiService {
     });
 
     this.router.post('/customers', (req, res) => {
-      stripe.customers.create(bodyToCustomer(req.body, 'source', req.user), defaultHandler(res));
+      try{
```

Comment:
```suggestion
      try {
```
Linter not trigger ?